src/c++/vdo/tests/OldRecoveryJournal_t1.c

```diff
@@ -141,6 +143,40 @@ static void generate(void)
   close(fd);
 }
 
+/**
+ * Allocate and read the VDO contents from disk into a character array.
+ **/
+static void readVDOFromDisk(const char *fileName)
+{
+  int fd;
+  VDO_ASSERT_SUCCESS(open_file(fileName, FU_READ_ONLY, &fd));
+
+  off_t vdoSize;
+  VDO_ASSERT_SUCCESS(get_open_file_size(fd, &vdoSize));
+
+  VDO_ASSERT_SUCCESS(vdo_allocate(vdoSize, char, __func__, &expectedData));
+  ssize_t bytesRead;
```

Comment:
This just looks awkward. This is a short enough function, can you pull all the local variable definitions up to the top of the function?

Reply:
Yeah. I had copied it from another test. Will fix.